### PR TITLE
Enable to set headers for remote url on #upload_attachment

### PR DESCRIPTION
**Pull request by nownabe**
(\\( •̀ө•́)/)

```rb
client = Esa::Client.new(access_token: "xxx", current_team: "xxx")
token = "xxx"
url = "https://xxx.qiita.com/files/xxx.png"
client.upload_attachment([url, { Authorization: "Bearer #{token}" }])
```

### CODE REVIEW

**Comment by ppworks** [comment]
:eyes:

File: lib/esa/api_methods.rb [hunk 1]
```diff
@@ -169,14 +169,18 @@ def content_type_from_file(file)
...
 
       if path_or_file_or_url.respond_to?(:read)
         path_or_file_or_url
       elsif path_or_file_or_url.is_a?(String) && HTTP_REGEX.match(path_or_file_or_url)
         remote_url = path_or_file_or_url
-        headers = {}
-        headers[:Cookie] = cookie if cookie
+        headers =
+          if headers_or_cookie
+            headers_or_cookie.is_a?(Hash) ? headers_or_cookie : { Cookie: headers_or_cookie }
```
Comment (ppworks) [inline lib/esa/api_methods.rb]: good 👍

**Comment by fukayatsu** [comment]
@nownabe Thanks!

**Comment by fukayatsu** [comment]
Published as 💎 v1.9.0

**Pull request merged**